ultimarc/devices/usb_button.py

```diff
@@ -50,16 +48,16 @@ def set_color(self, red, green, blue):
                 raise ValueError(_('Color argument value is invalid'))
 
         data = USBButtonColorStruct(0x01, red, green, blue)
-        return self.write(USBB_SET_COLOR, USBButtonReportID, USBButtonWIndex, data, ct.sizeof(data))
+        return self.write(USBRequestCode.SET_CONFIGURATION, 0x0, USBButtonWIndex, data, ct.sizeof(data))
```

Comment:
Change back to using USBButtonReportID here for the 0x0.

Reply:
Done